Formula/yarn.rb

```diff
@@ -17,7 +15,7 @@ class Yarn < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    cp_r '.', prefix
```

Comment:
What's the line after left in for? It appears to be a no-op (and I'm surprised it didn't fail).